From 683912a0731ddbd6364f3b6ca359be753f7d820a Mon Sep 17 00:00:00 2001
From: nferre <nferre@50fbe906-d41e-0410-8a96-31537896a350>
Date: Wed, 14 Oct 2009 13:31:46 +0000
Subject: [PATCH] AT91: correct AC97 reset line in at91sam9263ek board

Board code was wrongly setting up the reset pin for AC97 on at91sam9263ek.

Signed-off-by: Nicolas Ferre <nicolas.ferre@atmel.com>

git-svn-id: svn://rfolxts01.rfo.atmel.com/at91_sandbox/linux-2.6.x/branches/linux-2.6.30-at91@11612 50fbe906-d41e-0410-8a96-31537896a350
---
 arch/arm/mach-at91/board-sam9263ek.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-at91/board-sam9263ek.c b/arch/arm/mach-at91/board-sam9263ek.c
index 4cb1b1c..dd3f093 100644
--- a/arch/arm/mach-at91/board-sam9263ek.c
+++ b/arch/arm/mach-at91/board-sam9263ek.c
@@ -365,9 +365,9 @@ static void __init ek_add_device_buttons(void) {}
 
 /*
  * AC97
+ * reset_pin is not connected: NRST
  */
 static struct ac97c_platform_data ek_ac97_data = {
-	.reset_pin	= AT91_PIN_PA13,
 };
 
 
-- 
1.5.6.5

